apps/web/pages/api-access.tsx

```diff
@@ -1,10 +1,81 @@
+import {TabsPrimitive as Tabs} from '@usevenice/ui'
+import type {GetServerSideProps} from 'next'
+import {
+  DatabaseAccessCard,
+  PrimaryTabs,
+  QueryExplorer,
+} from '../components/api-access'
 import {PageHeader} from '../components/PageHeader'
 import {PageLayout} from '../components/PageLayout'
 
-export default function ApiAccessPage() {
+// for server-side
+import {z} from '@usevenice/util'
+import {getDatabaseInfo, serverGetUser} from '../server'
```

Comment:
next auto-magically removes server-only imports from the client bundle

from client bundle `/_next/static/chunks/pages/api-access.js`
<img width="900" alt="image" src="https://user-images.githubusercontent.com/3942264/218133869-95e5d71e-fc0b-43b3-aaa8-1f2de99f7728.png">